apps/ui/src/views/account/AccountProfileView.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <div>
+    <div v-if="userQuery.loading.value" class="loading">
+      <v-progress-circular color="#d6001c" indeterminate :size="100" :width="5">
+        <p class="text-center" style="color: white" >Loading Profile...</p>
+      </v-progress-circular>
+    </div>
+    <div v-else-if="!userData.person">
+      <v-form ref="profileCreation">
+        <p>You currently do not have a profile, you can create it by entering your name below.</p>
+        <v-text-field :rules="[nameRule]" label="Name" v-model="profileName" />
```

Comment:
Minor acessibility issue: Pressing enter here just refreshes the page instead of calling `createProfile`

Reply:
Fixed